Makefile

```diff
@@ -7,7 +7,8 @@ install: fmt
 	go install ./...
 
 clean:
-	rm -rf hostdir release whitepaper.aux whitepaper.log whitepaper.pdf sia.wallet sia/test.wallet sia/hostdir* sia/renterDownload
+	rm -rf hostdir release whitepaper.aux whitepaper.log whitepaper.pdf         \
+	sia.wallet sia/test.wallet sia/hostdir* sia/renterDownload
```

Comment:
I believe that after you start a line, all whitespace will be ignored. So I'll fix this.